docs/cert-manager.md

```diff
@@ -0,0 +1,462 @@
+# How-To: Deploy web applications on Kubernetes with Heptio Contour and Let's Encrypt
+
+This is a technical how-to that walks you through deploying a web application securely on a Kubernetes Cluster using HTTPS.
+The technologies we're going to use in this how-to are:
+
+- Kubernetes (obviously :))
+- Contour as the ingress controller
+- JetStack's cert-manager to provision TLS certificates from the Let's Encrypt project
+
```

Comment:
Link to Let's Encrypt here? Or to JetStack?

Reply:
Why not both :)